v1/fs/index.ts

```diff
@@ -32,6 +32,7 @@ export class SiteFileSystem {
         writable: false
       })
     )
-    return rimraf(tarballPath)
+    await rimraf(tarballPath)
+    return await Promise.resolve(sitePath)
```

Comment:
Should be able to just return `sitePath`. Is there a use for wrapping in an extra Promise?